metadata/service/aggregator/cluster.yml

```diff
@@ -0,0 +1,13 @@
+applications:
```

Comment:
you may want to factorize these classes by creating a aggregator/init.yml with common params:

The resulting class metadata/service/aggregator/cluster.yml will looks lighter:

```
classes:
- service.heka.aggregator
- service.heka.support
parameters:
  heka:
    aggregator:
      automatic_starting: false
```